### DISCUSSION TITLE
Copyright-assignment dropped

### DISCUSSION THREAD

**Discussion by huwdp** [Rust-GCC/gccrs#516] [General]
Hello, long time follower here.

Don't know if you know this but I just wanted to let you know that GCC has dropped assigned copyright-assignment so integrating Rust-GCC into GCC should be easier. Link here to article (https://lwn.net/Articles/857791/).

Says here (https://github.com/Rust-GCC/Reporting/commit/8c2c1a052f7e52ef87a98dc3cc0dea27fdb985d1) that "Be up front on all PRs that the code is destined to be upstreamed to FSF" under Copyright assignments section.

This is one big hurdle out of the way :)

**Comment by philberty** [comment]
Thanks, @huwdp we have an open issue: https://github.com/Rust-GCC/gccrs/issues/461. I have downgraded the likelihood of issues around copyright assignment from a 5 down to a 3 since the announcement. Though personally, I just want to let the dust settle for a while before changing anything with our contributing guidelines, so I would expect that this risk will be changed/removed from my reports soon.

Also thanks for reading my status reports, I put a lot of work into them to be open and transparent as I can be about the development process.
